### PR TITLE
Fix revdeps for Lwt 2.5.0

### PR DESCRIPTION
**Pull request by vbmithr**

### CODE REVIEW

File: packages/ketrew/ketrew.1.0.0/opam [hunk 1]
```diff
@@ -20,6 +20,6 @@ depends: [
   "trakeva" "sqlite3" "sosa" "nonstd" "docout" "pvem" "pvem_lwt_unix"
   "cmdliner" "yojson" "uri"
   "ppx_deriving" "ppx_deriving_yojson" {>= "2.3"} "ppx_include" "ppx_blob"
-  "cohttp" {= "0.17.2"} "lwt" "ssl"
+  "cohttp" {= "0.17.2"} "lwt" {< "2.5.0} "ssl"
```
Comment (dsheets) [inline packages/ketrew/ketrew.1.0.0/opam]: `{< "2.5.0"}`

**Comment by samoht** [comment]
```
===== ERROR while installing biocaml.0.3.0 =====
# opam-version 1.1.1
# os           linux
# command      omake -j2
# path         /home/travis/.opam/system/build/biocaml.0.3.0
# compiler     system (4.00.1)
# exit-code    2
# env-file     /home/travis/.opam/system/build/biocaml.0.3.0/biocaml-1281-2913ab.env
# stdout-file  /home/travis/.opam/system/build/biocaml.0.3.0/biocaml-1281-2913ab.out
# stderr-file  /home/travis/.opam/system/build/biocaml.0.3.0/biocaml-1281-2913ab.err
### stdout ###
# ...[truncated]
# User defined PREFIX: /home/travis/.opam/system
# *** omake: done (0.07 sec, 0/0 scans, 1/1 rules, 10/230 digests)
# *** omake: reading OMakefiles
# conf: Building biocaml application
# *** omake: finished reading OMakefiles (0.04 sec)
# --- Checking if ocamldep understands -modules... (yes)
# - build _build/app biocaml_app_common.o
# + ocamlfind ocamlc -syntax camlp4o -package core,threads,zip,unix,xmlm,pcre,cfstream,sexplib.syntax,flow -bin-annot -annot -warn-error P -thread -I . -I ../lib -c biocaml_app_common.ml
# *** omake: 407/444 targets are up to date
# *** omake: failed (49.97 sec, 98/98 scans, 140/191 rules, 385/1002 digests)
### stderr ###
# *** omake: warning: stdout is not a tty, disabling the progress bar
#    (use --progress to override).
# File "biocaml_app_common.ml", line 19, characters 2-15:
# Error: Unbound value wrap_deferred
# *** omake: targets were not rebuilt because of errors:
#    _build/app/biocaml_app_common.cmi
#    _build/app/biocaml_app_common.cmo
#    _build/app/biocaml_app_common.cmx
#    _build/app/biocaml_app_common.o
```
@agarwal any idea where's this could come from?

**Comment by dsheets** [comment]
#4399

**Comment by samoht** [comment]
can you rebase?

**Comment by dsheets** [comment]
Also #4400 for the flow errors which is still pending

**Comment by dsheets** [comment]
```
# File "_none_", line 1:
# Error: I/O error: No space left on device
# *** omake error:
#    Sys_error: No space left on device
```

is a new Travis error for me. There also appears to be something wrong with camlp4 on OS X (still? was fixed in homebrew maybe?). I've restarted the errored builds but otherwise we should merge this soon (as well as the lwt 2.5.0 release) and then clean up whatever breaks as it happens (perhaps with some nice bulk build runs on our developing test infrastructure).

**Comment by dsheets** [comment]
All remaining errors are timeouts or the trakeva OS X sqlite bug (#50!). Merging.

**Pull request merged**